apps/antalmanac/src/stores/ThemeStore.ts

```diff
@@ -0,0 +1,43 @@
+import { create } from 'zustand';
+import analyticsEnum, { logAnalytics } from '$lib/analytics';
+
+export interface ThemeStore {
+    /**
+     * The 'raw' theme (either 'light', 'dark', or 'system')
+     */
+    value: string;
+    /**
+     * The 'derived' theme (either 'light' or 'dark', based on user settings and device preferences)
+     */
+    theme: string;
```

Comment:
I think that `themeSetting` vs. `computedTheme` or `derivedTheme` makes better distinction than `theme` and the generic `value`. It can also be an enum.

Reply:
Looking at the interface alone, I wouldn't know if `setTheme` accepts `"system"`.